lib/cisco_node_utils/cmd_ref/interface_evpn_multisite.yaml

```diff
@@ -0,0 +1,12 @@
+# interface_evpn_multisite
+---
+_exclude: [ios_xr, N3k, N5k, N6k, N7k, N9k-F]
+
+_template:
+  get_command: "show running"
```

Comment:
If you have to use "show running-config" then it is fine but at least use the same command (here it is "show running"). There is no difference between these two at the platform level but our cache is per command based and both of these are seen as 2 different ones and so it will not use cache, which is not desirable.

Reply:
Modified this to 'show running interface all' based on the fact that all the other interface cmd_ref modules use that command.